docs/developer/github-actions.md

```diff
@@ -8,7 +8,9 @@ from [docker hub](hub.docker.com):
 | `DOCKERHUB_SCOPE` | Username or organization name on docker hub |
 | `DOCKERHUB_USERNAME` | Username on docker hub |
 | `DOCKERHUB_TOKEN` | API token to publish images to docker hub, with `Read`, `Write` and `Delete` permissions |
+| `NPM_TOKEN` | Token to publish npm package to Nodejs registry. |
+| `NPM_GITHUB_TOKEN` | Token to publish npm package to Github registry |
```

Comment:
Is this token necessary?

Reply:
I removed the NPM_GITHUB_TOKEN, since GITHUB_TOKEN can be used. But NPM_TOKEN is required for node js npm